backend/internal/service/handler/spotify/fetch_albums.go

```diff
@@ -0,0 +1,105 @@
+package spotify
+
+import (
+	"platnm/internal/models"
+	"platnm/internal/service/ctxt"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/zmb3/spotify/v2"
+)
+
+type AlbumArtist struct {
+	AlbumID  int
+	ArtistID int
+}
+
+type AddedContent struct {
+	Albums  []*models.Album
+	Artists []*models.Artist
+}
+
+func (h *SpotifyHandler) GetNewReleases(c *fiber.Ctx) error {
+	newReleases, err := fetchNewReleasesFromSpotify(c)
+	if err != nil {
+		return err
+	}
+
+	addedContent := &AddedContent{}
+	for _, album := range newReleases.Albums {
+		if err := h.handleAlbum(c, album, addedContent); err != nil {
+			return err
+		}
+	}
+
+	return c.Status(fiber.StatusOK).JSON(addedContent)
+}
+
+func (h *SpotifyHandler) handleAlbum(c *fiber.Ctx, album spotify.SimpleAlbum, addedContent *AddedContent) error {
```

Comment:
```suggestion
func (h *SpotifyHandler) handleAlbum(ctx context.Context, album spotify.SimpleAlbum, addedContent *AddedContent) error {
```
then change arg when called from `c` ->  `c.Context()`

Reply:
since you are only concerned with the `c.Context()` anyways